### DISCUSSION TITLE
Mouse Follow

### DISCUSSION THREAD

**Discussion by GamingJakob** [rustdesk/rustdesk#2662] [General]
I would like to see an option added to follow the other mouse pointer.
If this option is active, the remote monitor is always displayed where the mouse of the remote computer is located, or the monitor on which the user is currently working.

**Comment by xugj-gits** [comment]
这是来自QQ邮箱的假期自动回复邮件。请关注淘宝：http://shop64118300.taobao.com/您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
